src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -317,6 +319,10 @@ public MethodField makeMethodField(String name, List<SQLExpr> arguments, SQLAggr
             } else if (object instanceof SQLCastExpr) {
                 String scriptCode = new CastParser((SQLCastExpr) object, alias, tableAlias).parse(false);
                 paramers.add(new KVValue("script", new SQLCharExpr(scriptCode)));
+            } else if (object instanceof SQLAggregateExpr) {
```

Comment:
Does this also cover other cases such as `MAX(age + 1)`, `MAX(LOG(age))`? If not, any chance to take care of them too?

Reply:
> MAX(age + 1),

This is quite different case, out of scope of this PR

> MAX(LOG(age))
surprisingly, this is **almost** working, I think it should be just fixed